.rtd-env-py3.yml

```diff
@@ -5,6 +5,7 @@ channels:
 dependencies:
 - python=3.8
 - cantera
+- pip
```

Comment:
Are you sure this isn't redundant?

Reply:
Without this line, the build warns about a missing pip dependency. See e.g. the log at https://readthedocs.org/projects/pyrometheus/builds/19198872/, under the `conda env create ...` section.